### DISCUSSION TITLE
There is no extension able to load the configuration for "pimcore_enterprise_subscription_tools

### DISCUSSION THREAD

**Discussion by IMCtoys** [pimcore/pimcore#16117] [Q&A / Support] [answered]
Hi,

we have the enterprise license, and followed the steps to install the enterprise extensions:

- Add token authentication to composer
- Add respository to composer

But when we try to install any enterprise extension we receive this error:

"Exit code: 255

  Symfony\Component\Config\Exception\LoaderLoadException {#1490
    #message: "There is no extension able to load the configuration for "pimcore_enterprise_subscription_tools" (in "/var/www/html/pimcoreproject/config/packages/dev/../../config.yaml")"

I have seen a previous ticket with the same error, but the user said it was resolved without additional info. 

Could you please kindly help?

thank you

**Comment by NiklasBr** [comment, marked as answer]
The documentation is missing a crucial step, you should also install the bundle: `composer require  pimcore/enterprise-subscription-tools`

**Comment by IMCtoys** [comment]
Great! it worked! Thank you very much Niklas!
